README.md

```diff
@@ -278,6 +278,11 @@ This effector is useful when the `tf.configuration.url` points to a dynamic URL,
 
 **Note** Invoking the `reinstallConfig` effector will not affect the `*.tfvars` file that is provided using the `tf.tfvars.url` configuration key.
 
+#### Using the Managed Resource effectors
+
+The Managed Resources entities (children of the `TerraformConfiguration`) have the default effectors for a Startable type entity, i.e. `start`, `stop` and `restart`. 
+Currently (as of AMP 6.1), these effectors have no functionality for the Managed Resources and are reserved for future use. Invoking these effectors currently has no effect on the entity, which the effector is invoked for.
```

Comment:
Delete reference to AMP here

Reply:
Also change last sentence in paragraph to remove duplicate use of 'currently', maybe reword completely to:
> Invoking any of these effectors will have no effect on the entity.